core/src/main/java/org/elasticsearch/script/StoredScriptSource.java

```diff
@@ -425,6 +425,11 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
 
         return builder;
     }
+    
+    @Override
+    public boolean isFragment() {
```

Comment:
implement ToXContentObject instead? I wish it would be possible to make default methods final :) I would fix this on master too. Not a huge deal but I think it's more correct this way.